### ISSUE TITLE
Version 7.1.0 has syntax error

### ISSUE THREAD

**Issue by middagj** [minio/charts#34]
c5733b5a gives

```
Error: parse error at (minio/templates/deployment.yaml:99): unexpected {{end}}
```

If I check the commit, I see an _if_ and an _else_ removed, but no _end_.

**Comment by mboutet** [comment]
Two `{{ end }}` need to be removed at https://github.com/minio/charts/blob/c5733b5abbf459b33126a31e7fe134adf9f8c4d2/minio/templates/deployment.yaml#L94-L95

**Comment by harshavardhana** [comment]
Fixed by 76372e091dd632beb2eab2617f0be5b55548d518
